google/cloud/automl_v1beta1/services/tables/tables_client.py

```diff
@@ -1282,7 +1330,6 @@ def update_column_spec(
             column_spec_display_name=column_spec_display_name,
             project=project,
             region=region,
-            **kwargs
```

Comment:
Why are these kwargs deleted? I am fine with not passing (deleting) the kwargs, just want to understand what's the convention we want to set. Is the conversion kwargs is only used by the wrapped method? E.g., kwargs argument of `update_column_spec` is only used by `self.auto_ml_client.update_column_spec`.

Reply:
I think that's the convention we're going to have to set. The new surface doesn't allow passing along arbitrary kwargs, because method kwargs have to be split off from request kwargs.

I'll take another pass to make this consistent throughout the file.

Reply:
Updated to consistently follow your suggestion (kwargs argument of `update_column_spe`c is only used by `self.auto_ml_client.update_column_spec`) and noted in the migration guide. PTAL.